streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesSessionClientV2.scala

```diff
@@ -133,4 +165,115 @@ object KubernetesSessionClientV2 extends KubernetesClientV2Trait with Logger {
     }
   }
 
+  private def genFlinkConfig(deployRequest: DeployRequest): Configuration = {
+    val flinkConfig =
+      extractConfiguration(deployRequest.flinkVersion.flinkHome, deployRequest.properties)
+    flinkConfig
+      .safeSet(KubernetesConfigOptions.NAMESPACE, deployRequest.k8sDeployParam.kubernetesNamespace)
+      .safeSet(
+        KubernetesConfigOptions.KUBERNETES_SERVICE_ACCOUNT,
+        deployRequest.k8sDeployParam.serviceAccount)
+      .safeSet(
+        KubernetesConfigOptions.REST_SERVICE_EXPOSED_TYPE,
+        ServiceExposedType.valueOf(deployRequest.k8sDeployParam.flinkRestExposedType.getName))
+      .safeSet(KubernetesConfigOptions.CLUSTER_ID, deployRequest.clusterId)
+      .safeSet(KubernetesConfigOptions.CONTAINER_IMAGE, deployRequest.k8sDeployParam.flinkImage)
+      .safeSet(
+        KubernetesConfigOptions.KUBE_CONFIG_FILE,
+        getDefaultKubernetesConf(deployRequest.k8sDeployParam.kubeConf))
+      .safeSet(DeploymentOptionsInternal.CONF_DIR, s"${deployRequest.flinkVersion.flinkHome}/conf")
```

Comment:
Remove these dangerous configs  `KubernetesConfigOptions.KUBE_CONFIG_FILE` and `DeploymentOptionsInternal.CONF_DIR`  for flink kubernetes operator